### PR TITLE
Allow DOTNET_SSL_DIRS to be volumes mounted into the application cont…

### PR DESCRIPTION
**Pull request by tmds**
This enables DOTNET_SSL_DIRS to include volumes that are mounted into the container.
Before this changes, only directories that were part of the build container ould be used as DOTNET_SSL_DIRS.

Implements https://github.com/redhat-developer/s2i-dotnetcore/issues/213.

### CODE REVIEW

File: 2.1/runtime/README.md [hunk 1]
```diff
@@ -76,11 +76,11 @@ They must not to be overridden.
...
     These variables contain the working directory (`/opt/app-root/app`) and default CMD of the runtime image (`default-cmd.sh`).
 
-* **DOTNET_SSL_CERT_DIR**
+* **DOTNET_SSL_DIRS**
 
-    This variable is set to `/opt/app-root/ssl_dir`. This directory is not present in the runtime image.
-    When it added (e.g. by s2i assemble), it is used as `SSL_CERT_DIR`. `SSL_CERT_DIR` is used by applications (including `dotnet`)
-    to load CA certificates.
+    Used to specify a list of folders with additional certificates to trust. The certificates are trusted by each process that runs
+    during the build and all processes that runs in the image after the build (including the application that was built). The folders
```
Comment (aslicerh) [inline 2.1/runtime/README.md]: Run here should be singular.

**Comment by aslicerh** [comment]
As you said, I'm not worried about these changes breaking undocumented behavior. That said, if a user needed to add certs for use at runtime, but wanted to rm src, is there a way to do that currently?

**Comment by tmds** [comment]
> That said, if a user needed to add certs for use at runtime, but wanted to rm src, is there a way to do that currently?

The user should publish the certificates with his application and include the published certificate directory in DOTNET_SSL_DIRS.

**Comment by tmds** [comment]
Thanks for reviewing @aslicerh !

**Pull request merged**